test/functional/page_objects/visual_builder_page.ts

```diff
@@ -516,7 +516,9 @@ export class VisualBuilderPageObject extends FtrService {
 
   public async checkIndexPatternSelectionModeSwitchIsEnabled() {
     await this.toggleIndexPatternSelectionModePopover();
-    return await this.testSubjects.isEnabled('switchIndexPatternSelectionMode');
+    const isEnabled = await this.testSubjects.isEnabled('switchIndexPatternSelectionMode');
+    await this.toggleIndexPatternSelectionModePopover();
```

Comment:
the following code not looks professional. I prefer to write code in more controllable way. Please pass `toggle` argument into toggleIndexPatternSelectionModePopover
```

await this.toggleIndexPatternSelectionModePopover();
..
await this.toggleIndexPatternSelectionModePopover();
```